rollup.config.js

```diff
@@ -20,12 +19,16 @@ export default [
     input: 'src/index.js',
     output: [output('ESMWebexSDKComponentAdapter', 'esm')],
     plugins: [
-      resolve(),
+      resolve({preferBuiltins: false}),
       babel({
         runtimeHelpers: true,
       }),
-      commonJS(),
+      commonJS({
+        namedExports: {
+          '@webex/common': ['deconstructHydraId'],
+        },
+      }),
     ],
-    external: ['@webex/common', 'rxjs', 'rxjs/operators', 'webex'],
+    external: ['webex', 'rxjs', 'rxjs/operators'],
```

Comment:
Nit-picky but we were trying to keep the modules in alphabetical order

Reply:
okay I'll update